dbt_project.yml

```diff
@@ -17,8 +17,8 @@ clean-targets:         # directories to be removed by `dbt clean`
   - "dbt_packages"
 
 dispatch:
-  - macro_namespace: dbt_utils
-    search_order: ['dbt_project_evaluator', 'spark_utils', 'dbt_utils']
+  - macro_namespace: dbt
+    search_order: ['dbt_project_evaluator', 'dbt']
```

Comment:
If you dispatch the macros below to your own package (as discussed) then you won't need this at all - as you identified, it feels wrong because it is!

Reply:
roger that captain!